solidity/test/DepositFundingTest.js

```diff
@@ -125,6 +130,27 @@ describe("DepositFunding", async function() {
       )
     })
 
+    it("reverts if bond is insufficient to cover a deposit creation fee refund", async () => {
+      const depositFee = await tbtcSystemStub.getNewDepositFeeEstimate()
+
+      await ecdsaKeepStub.send(depositFee - 1)
+      await ecdsaKeepStub.setBondAmount(depositFee - 1)
```

Comment:
Head-whirly that we don't have to mess with the ETHBTC price here, but more illustrative of how complicated all the bits are.

Reply:
yeah it's definitely gotten incrementally more complicate